### ISSUE TITLE
[Feature Request] Formatted .show files.

### ISSUE THREAD

**Issue by Snack-K** [ChurchApps/FreeShow#965]
Hi!

I have imported my songbook from OpenSong. My songs' first couple of slides were sheet images. FreeShow did not imported the images from the .xml files so I tried to write a script which locate my images and pair them to the correct .show files and add the image source path into them. I can't do it because everything is in one line which would be pretty hard to parse.

It would be nice if .show files would be in readable format or FreeShow would import the images from OpenSong .xml files.

An example from one of my OpenSong file:
`<?xml version="1.0"?>`
`<song><title>Some title</title><presentation>V1A V1B V2 V3 V4</presentation>`
`<lyrics>`
`[V1A]`
`1-1`
`[V1B]`
`1-2`
`[V1]`
`Some lyrics`
`[V2]`
`Some lyrics`
`</lyrics>`
`<backgrounds resize="body" keep_aspect="true" link="false" background_as_text="true"><background verse="V1A"><image>/9j/4AAQSkZJRgABAQAAAQABAAD/...`

The image section would be too long to paste it here. Anyway, FreeShow imports [V1A] and [V1B] with the text 1-1 and 1-2.

**Comment by vassbo** [comment]
The .show files are in JSON which is a very readable format and more parsable than XML, but you might need to use an IDE for easier reading such as code formatting and syntax highlighting.

You could also send an example OpenSong file with a background and I'll add support for it!
